universal-login-react/src/ui/ConnectionFlow/ConnectionFlow.tsx

```diff
@@ -21,10 +21,17 @@ export interface ConnectionFlowProps {
 
 export const ConnectionFlow = ({basePath = '', name, onCancel, onSuccess, sdk, walletService, className}: ConnectionFlowProps) => {
   const history = useHistory();
+
+  const onConnectWithDeviceClick = () => {
+    walletService.initializeConnection(name).then(() => {
+      history.push('/connect/emoji');
+    });
```

Comment:
It is not important, but i would like:
walletService.initializeConnection(name).then(() => history.push('/connect/emoji'));